CHANGELOG.md

```diff
@@ -1,8 +1,46 @@
 <!--markdownlint-disable MD012 no-multiple-blanks -->
 <!--markdownlint-disable MD024 no-duplicate-heading/no-duplicate-header -->
 
+# 0.57.0
+
+> Released 2023-04-10
+
+## Added
+
+* Whiskey's verbose and debug messages now include timing information like information messages.
+* Parameter `PipelineName` to Whiskey's default build.ps1 script, which allows running a specific pipeline from a
+whiskey.yml file.
+* Parameter `ConfigurationPath` to Whiskey's default build.ps1 script, which allows running a build using a specific
+whiskey.yml file.
+* The `PublishProGetAsset` and `PublishProGetUniversalPackage` tasks now write what they're publishing to the
+information stream.
+
+## Changed
+
+* Updated the `Pester` task to no longer run tests in a background job.
+[There's about a 6x reduction in the stack size of PowerShell processes started by `Start-Job`](https://github.com/PowerShell/PowerShell/issues/17407),
+which causes "call depth overflow" exceptions during some Pester tests. The `Pester` task now runs tests in a
+seperate PowerShell process.
+* Updated the `ProGetUniversalPackage`, `PublishProGetAsset`, `PublishProGetUniversalPackage` tasks to use the latest
+ProGetAutomation version that matches wildcard `1.*` (excluding prerelease versions). They were using version `0.10.*`.
+* Updated `Version` task to no longer use a privately packaged version of ProGetAutomation and instead use the same
+version as the `ProGetUniversalPackage`, `PublishProGetAsset`, and `PublishProGetUniversalPackage` tasks.
+
+## Fixed
+
+* Fixed: Installing .NET fails if the global.json file requests a version of .NET that is newer than any installed

```

Comment:
Also fixed: Installing .NET fails when the `global.json` file does not contain a `rollForward` property.